### PR TITLE
Always use the API for inline scripts

### PR DESCRIPTION
**Pull request by dgrammatiko**
#### Don't echo script tags in the body!

All inline scripts should go through Joomla's API!
#### Testing

Apply the PR with patchtester
Go to `administrator/index.php?option=com_modules`
display the search tools
and try the dropdown for position, module name and language.
If the filtering works fine then you have a successful test

### CODE REVIEW

File: administrator/components/com_modules/layouts/joomla/searchtools/default/bar.php [hunk 1]
```diff
@@ -12,12 +12,15 @@
 $data = $displayData;
 
 $clientIdField = $data['view']->filterForm->getField('client_id');
+JFactory::getDocument()->addScriptDeclaration(
+<<<JS
```
Comment (andrepereiradasilva) [inline administrator/components/com_modules/layouts/joomla/searchtools/default/bar.php]: what's "<<<JS" ... "JS"?

Comment (dgt41) [inline administrator/components/com_modules/layouts/joomla/searchtools/default/bar.php]: Check https://secure.php.net/manual/en/function.echo.php and search for `<<<END`
So this is standard PHP notation with a benefit to use both single and double quotes and also the script will be nicely formatted in your IDE.
I consider proposing to standardise this for inline scripts

Comment (andrepereiradasilva) [inline administrator/components/com_modules/layouts/joomla/searchtools/default/bar.php]: always learning :wink:

Comment (dgt41) [inline administrator/components/com_modules/layouts/joomla/searchtools/default/bar.php]: Here is what we get in the IDE with both ways:
Colons:
screen shot 2016-01-17 at 18 11 06
<<<JS
screen shot 2016-01-17 at 18 11 24

**Comment by 810** [comment]
I have tested this item :white_check_mark: successfully on 861997d82a514361c92047c9b42494500c26e025<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8920">issues.joomla.org/joomla-cms/8920</a>.</sub>

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on 861997d82a514361c92047c9b42494500c26e025<br /><br />Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8920">issues.joomla.org/joomla-cms/8920</a>.</sub>

**Comment by wilsonge** [comment]
Merged on review

**Pull request merged**